blog/docusaurus-zielony-przyjaciel/index.mdx

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Docusaurus - zielony przyjaciel tech writera'
+authors: pensjonatus
+date: '2024-05-28'
+tags:
+  - 'warsztat'
+  - 'narzędzia'
+coverImage: 'docusaurus-zielony-przyjaciel.png'
+---
+
+Generator stron statycznych stworzony z myślą o dokumentacji. Czy się faktycznie
+nadaje? Oto nasze wrażenia.
+
+{/* truncate */}
+
+## Co to jest Docusaurus?
+
+Docusaurus to Static Site Generator, czyli aplikacja do generowania stron
+statycznych. Przedstawicielem tej grupy jest na przykład DITA OT, ale jest ich
+[znacznie więcej](https://jamstack.org/generators/). Podobnie jak DITA OT,
+Docusaurus został stworzony do dokumentacji, ale nie ma on nic wspólnego z ditą.
+W Docusaurusie piszemy w formacie Markdown, a konkretnie MDX.
+
+W dużym skrócie, projekt strony z Docusaurusem ma folder "docs" w którym piszemy
+swoje strony (topiki, artykuły) i plik, który pozwala nam zarządzać nawigacją.
+
+Jeżeli korzystacie z narzędzi wokół programowania front-end, to na pewno
+widzieliście już Docusaurusa, na przykład na stronie
+[Yarn](https://yarnpkg.com/) albo
+[Testing Library](https://testing-library.com/). Jeżeli nie, to może
+zauważyliście, że w (w momencie publikacji tego artykułu) Docusaurus zasila
+strony [MeetContent](https://meetcontent.github.io/) i stronę, która aktualnie
+czytasz.
+
+## Dlaczego do dokumentacji?
+
+Docusaurus od początku był stworzony z myślą o dokumentacji do projektów open
+source dla firmy Meta (ówcześnie Google Open Source). Joel Marcey, ich
```

Comment:
Google Open Source? Meta to Facebook, nie wiedziałem, że Docusaurus ma coś wspólnego z Google 🤔